### PR TITLE
fix: altered the prescription and medicine models as M2M relationship

### PR DESCRIPTION
**Pull request by Kavindu-248**

### CODE REVIEW

File: apps/dispensary/models.py [hunk 1]
```diff
@@ -62,7 +61,17 @@ class Prescription (models.Model):
         'assesment.FormAssesment', on_delete=models.CASCADE, related_name='prescription', null=True, blank=True)
     appointment = models.OneToOneField(
         'assesment.Appointment', on_delete=models.CASCADE, related_name='prescription', null=True, blank=True)
-    medicines = models.ManyToManyField(Medicine, related_name='prescriptions')
+    medicines = models.ManyToManyField( Medicine, through='PrescriptionMedicine', related_name='prescriptions')
+
+
+# Prescription Medicine Model
+class PrescriptionMedicine(models.Model):
```
Comment (Raam124) [inline apps/dispensary/models.py:68]: check both FKs related names those are incorrect

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/dispensary/models.py (hunk 1 above)
Comment (Kavindu-248) [reply apps/dispensary/models.py:68]: Changed the relevant name

Comment (Raam124) [reply apps/dispensary/models.py:68]: these are still not relevant names just make sure you got how related names work

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/dispensary/models.py (hunk 1 above)
Comment (Raam124) [reply apps/dispensary/models.py:68]: this is still irrelevant

**Review by Raam124 — CHANGES_REQUESTED** [review]

**Review by Raam124 — CHANGES_REQUESTED** [review]

**Review by Raam124 — APPROVED** [review]

**Pull request merged**